schemas/com.snowplowanalytics.snowplow.media/player/jsonschema/1-0-0

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity for media events that describes the media player and playback state",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "player",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "currentTime": {
+      "type": "number",
+      "description": "The current playback time",
```

Comment:
`in seconds`? Should this mention "position" at all or is that more confusing (thinking `player.currentTime` sounds a lot like `sum(session.timePlayed)`)?

Reply:
Good point, the property is based on the [HTML5 player one](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/currentTime) but I agree that is a bit confusing considering the stats in session. Have changed the description to clarify it more and mention position.